### ISSUE TITLE
1.4-Pixel-Perfect-Aaron Emery

### ISSUE THREAD

**Issue by a-emery** [TIY-GVL-FEE-2015-August/assignments#65]
### Tasks

* **Getting Started**
    * [x] Follow the assignment instructions to create a new issue.
    * [x] Follow the assignment instructions to create a new repository.
    * [x] Create and commit `index.html`
    * [x] Follow the assignment instructions to create a new pull request.
    * [x] **Read [this article](http://diveintohtml5.info/semantics.html) about
      semantic HTML.**
* **Make It So**
    * [x] Checkpoint: Basic HTML structure committed!
    * [x] Checkpoint: Basic layout CSS committed!
    * [x] Checkpoint: Pixel perfect page header committed!
    * [x] Checkpoint: Pixel perfect page footer committed!
    * [x] Checkpoint: Pixel perfect column layout committed!
    * [x] Checkpoint: Pixel perfect layout and typography committed!
    * [x] Checkpoint: Links with hover states committed!
    * [x] Mission complete!

**Comment by a-emery** [comment]
Can be viewed at: http://a-emery.github.io/1.4-pixel-perfect/

Comfort zone: 4

Time worked: 12-13ish hours

Results: Finished. I'm fairly happy with the outcome. There are some areas that don't quite match up, but I think most of it is because of font issues. I liked this project because I think it is going to help hone my CSS and HTML abilities. By the end of the project I started feeling more confident in what I was doing. I'm sure compared to a good, proper coding job, this is a mess and has way more CSS than is needed, but I think if I made it again I would probably be a little more efficient. The comfort zone of 4 comes mostly from a few more specific details of the page, particularly getting the first hover state working and the three buttons after the article. The hovers got easier as I progressed through the project. One thing I wish I had done was make it more responsive from the start, particularly in the header. When I resized the browser the first time (unfortunately after spending hours on the header and body), the site broke horribly. Because of this, I set a fixed width of 1300px.

Repo link: https://github.com/a-emery/1.4-pixel-perfect
